README.md

```diff
@@ -26,58 +28,33 @@
 5. Install and update [Android Studio](https://developer.android.com/studio/index.html)
 * React Native Android installation [instructions](https://facebook.github.io/react-native/docs/getting-started.html)
 
-## Installation:
-
-1. Clone this repo
-
-2. In your terminal, change your directory to the root of the sample project you want to run.
-
-3. Run the command:
- 
-        npm install
-    to install required node modules
-
-4. For iOS:
-  * `cd ios`
-  * `pod install`
-
-4. For Android:
-  * `react-native link`
-  *  Add the following to your project's `build.gradle` file: 
+## Installation
 
-            maven {
-                url "http://tokbox.bintray.com/maven"
-            }
+To install this package in your project, run the following command:
```

Comment:
This is a sample project and shouldn't really be going into other projects as a dependency.

Reply:
That's right. 
I think we should add a link to [opentok-react-native](https://github.com/opentok/opentok-react-native) repo.
My suggestion would be after the "OpenTok Labs" banner. 

Something along the lines:

>    These samples demonstrate how to use opentock-react-native to do video-calling in a React Native application.